azure-pipelines.yml

```diff
@@ -479,63 +479,58 @@ stages:
+                TEST_INCLUDE: "**/sonarqube/*"
               vs2017_latest99:
                 PRODUCT: "SonarQube"
                 SQ_VERSION: "LATEST_RELEASE[9.9]"
                 SONAR_CFAMILYPLUGIN_VERSION: "6.41.0.60884"  # LATEST_RELEASE of CFAMILY is not compatible with old SQ
                 MSBUILD_PATH: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\MSBuild\\15.0\\Bin\\MSBuild.exe"
-                JDKVERSION: "1.17"
-                TEST_SUITE: "**/SonarQubeTestSuite.java"
+                TEST_INCLUDE: "**/sonarqube/*"
               vs2019_latest99:
                 PRODUCT: "SonarQube"
                 SQ_VERSION: "LATEST_RELEASE[9.9]"
                 SONAR_CFAMILYPLUGIN_VERSION: "6.41.0.60884"  # LATEST_RELEASE of CFAMILY is not compatible with old SQ
                 MSBUILD_PATH: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\MSBuild\\Current\\Bin\\MSBuild.exe"
-                JDKVERSION: "1.17"
-                TEST_SUITE: "**/SonarQubeTestSuite.java"
+                TEST_INCLUDE: "**/sonarqube/*"
               vs2022_latest99:
                 PRODUCT: "SonarQube"
                 SQ_VERSION: "LATEST_RELEASE[9.9]"
                 SONAR_CFAMILYPLUGIN_VERSION: "6.41.0.60884" # LATEST_RELEASE of CFAMILY is not compatible with old SQ
                 MSBUILD_PATH: $(MSBUILD_2022_PATH)
-                JDKVERSION: "1.17"
-                TEST_SUITE: "**/SonarQubeTestSuite.java"
+                TEST_INCLUDE: "**/sonarqube/*"
               vs2022_latest:
                 PRODUCT: "SonarQube"
                 SQ_VERSION: "LATEST_RELEASE"
                 SONAR_CFAMILYPLUGIN_VERSION: "LATEST_RELEASE"
                 MSBUILD_PATH: $(MSBUILD_2022_PATH)
-                JDKVERSION: "1.17"
-                TEST_SUITE: "**/SonarQubeTestSuite.java"
+                TEST_INCLUDE: "**/sonarqube/*"
               vs2022_dev:
                 PRODUCT: "SonarQube"
                 SQ_VERSION: "DEV"
                 SONAR_CFAMILYPLUGIN_VERSION: "LATEST_RELEASE"
                 MSBUILD_PATH: $(MSBUILD_2022_PATH)
-                JDKVERSION: "1.17"
-                TEST_SUITE: "**/SonarQubeTestSuite.java"
+                TEST_INCLUDE: "**/sonarqube/*"
               vs2022_sonar_cloud:
                 PRODUCT: "SonarCloud"
                 SQ_VERSION: ""
                 MSBUILD_PATH: $(MSBUILD_2022_PATH)
-                JDKVERSION: "1.17"
-                TEST_SUITE: "**/SonarCloudTestSuite.java"
+                TEST_INCLUDE: "**/sonarcloud/*"
           variables:
             SONAR_DOTNET_VERSION: 'DEV'
+            JDKVERSION: "1.17"
             PLATFORMTOOLSET: 'v140'
             WINDOWSSDKTARGET: '10.0.17763.0'
             MAVEN_CACHE_FOLDER: $(Pipeline.Workspace)/.m2/repository
             MAVEN_OPTS: '-Xmx3072m -Dmaven.repo.local=$(MAVEN_CACHE_FOLDER)'
+            ARTIFACTORY_NUGET_USERNAME: $(ARTIFACTORY_PRIVATE_READER_USERNAME)
```

Comment:
This is a bit weird.
Is there any reason that the name of the environment variable is different than [here](https://github.com/SonarSource/sonar-dotnet/blob/master/azure-pipelines.yml#L75)?
Wouldn't that mean that we will need to set 4 ENV variables instead of 2?

I suggest we rename this to be the same as in sonar-dotnet (both in here and in the Nuget.Config)

Reply:
This is the original state that I raised a point about, to unify the names.

Currently, this is the name as used on master. The rename PR was not merged yet see #1843, so I had to use the old name here. I also left a note there, that the rename PR will need to take this new line into account.